test-e2e/k8spod.py

```diff
@@ -14,3 +15,76 @@ def ping(self, ip):
             return False
         return True
 
+    def http(self, client_name):
+        httpd_cmd = '/bin/sh -c python3 -m http.server'
```

Comment:
Please use the same commands we previously had there https://github.com/futurewei-cloud/mizar/blob/dev-next/test/trn_controller/endpoint.py#L79-L87

Here the httpd ```pushd /tmp/{self.ns}_{self.ip}; python3 -m http.server > /tmp/{self.ns}_{self.ip}/httpd.log 2>&1```